### DISCUSSION TITLE
Error when I use prophet model and TFT, DeepAR, Simple model

### DISCUSSION THREAD

**Discussion by riksohet2018** [tinkoff-ai/etna#644] [Q&A]
Good day!
Could you tell me please I instaled !pip install Etna[all] and also the necessary libraries separately the version of which are registered with you, but I still get an error:

NameError: name 'Prophet' is not defined

**Comment by martins0n** [comment]
Hello!

Could you add output of `pip install etna[all]` and `pip freeze` after installation on fresh enviroment?

**Comment by LubinNK** [comment]
![image](https://user-images.githubusercontent.com/82504030/225584792-e3a67964-d2ae-4203-a9b3-f9a0c6826742.png)
![image](https://user-images.githubusercontent.com/82504030/225584989-b09f6f89-0ee6-40dc-8941-08e8fafd79cd.png)

absl-py==1.4.0
aiohttp==3.8.4
aiosignal==1.3.1
albumentations==1.3.0
alembic==1.10.2
antlr4-python3-runtime==4.9.3
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
astroid==2.12.9
asttokens==2.0.7
astunparse==1.6.3
async-timeout==4.0.2
attrs==22.1.0
autopage==0.5.1
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
boto3==1.26.91
botocore==1.29.91
Bottleneck==1.3.7
cachetools==5.3.0
catboost==1.1.1
certifi==2022.9.14
cffi==1.15.1
charset-normalizer==2.1.1
click==8.1.3
cliff==4.2.0
cmaes==0.9.1
cmd2==2.4.3
cmdstanpy==1.1.0
colorama==0.4.5
colorlog==6.7.0
contourpy==1.0.5
convertdate==2.4.0
coverage==6.5.0
cycler==0.11.0
Cython==0.29.33
debugpy==1.6.2
decorator==5.1.1
defusedxml==0.7.1
Deprecated==1.2.13
dill==0.3.5.1
docker-pycreds==0.4.0
emoji==2.2.0
entrypoints==0.4
ephem==4.1.4
etna==1.15.1
exceptiongroup==1.0.1
executing==0.10.0
fastjsonschema==2.16.1
flake8==5.0.4
flake8-for-pycharm==0.4.1
flatbuffers==23.3.3
fonttools==4.37.3
frozenlist==1.3.3
fsspec==2023.3.0
gast==0.4.0
gitdb==4.0.10
GitPython==3.1.31
google-auth==2.16.2
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
graphviz==0.20.1
greenlet==2.0.2
grpcio==1.51.3
h5py==3.7.0
hijri-converter==2.2.4
holidays==0.13
hydra-slayer==0.2.0
idna==3.4
imageio==2.22.4
importlib-metadata==6.0.0
iniconfig==1.1.1
ipykernel==6.15.1
ipython==8.4.0
ipython-genutils==0.2.0
ipywidgets==7.7.3
isort==5.10.1
jedi==0.18.1
Jinja2==3.1.2
jmespath==1.0.1
joblib==1.2.0
jsonschema==4.9.1
jupyter-client==7.3.4
jupyter-core==4.11.1
jupyterlab-pygments==0.2.2
jupyterlab-widgets==1.1.2
keras==2.11.0
kiwisolver==1.4.4
korean-lunar-calendar==0.3.1
lazy-object-proxy==1.7.1
libclang==15.0.6.1
lightning-utilities==0.8.0
llvmlite==0.38.1
loguru==0.5.3
LunarCalendar==0.0.9
lxml==4.9.1
Mako==1.2.4
Markdown==3.4.1
MarkupSafe==2.1.1
matplotlib==3.6.0
matplotlib-inline==0.1.3
mccabe==0.7.0
mistune==0.8.4
mlxtend==0.21.0
multidict==6.0.4
mypy==0.991
mypy-extensions==0.4.3
mysql-connector-python==8.0.30
nbclient==0.6.6
nbconvert==6.5.3
nbformat==5.4.0
nest-asyncio==1.5.5
networkx==2.8.8
notebook==6.4.12
numba==0.55.2
numpy==1.22.4
oauthlib==3.2.2
omegaconf==2.3.0
opencv-python-headless==4.6.0.66
opt-einsum==3.3.0
optuna==2.10.1
packaging==21.3
pandas==1.5.1
pandocfilters==1.5.0
parso==0.8.3
pathtools==0.1.2
patsy==0.5.3
pbr==5.11.1
pickleshare==0.7.5
Pillow==9.2.0
platformdirs==2.5.2
plotly==5.13.1
pluggy==1.0.0
pmdarima==2.0.2
prettytable==3.6.0
prometheus-client==0.14.1
promise==2.3
prompt-toolkit==3.0.30
prophet==1.1
protobuf==3.19.6
psutil==5.9.1
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.9.1
pycparser==2.21
pyflakes==2.5.0
Pygments==2.12.0
pylint==2.15.2
PyMeeus==0.5.12
pyparsing==3.0.9
pyperclip==1.8.2
pyreadline3==3.4.1
pyrsistent==0.18.1
pytest==7.2.0
pytest-cov==4.0.0
python-dateutil==2.8.2
pytorch-forecasting==0.9.2
pytorch-lightning==1.9.4
pyts==0.12.0
pytz==2022.5
PyWavelets==1.4.1
pywin32==304
pywinpty==2.0.7
PyYAML==6.0
pyzmq==23.2.1
qudida==0.0.4
requests==2.28.1
requests-oauthlib==1.3.1
rsa==4.9
ruptures==1.1.5
s3transfer==0.6.0
scikit-image==0.19.3
scikit-learn==1.0.2
scikit-plot==0.3.7
scipy==1.7.3
seaborn==0.11.2
Send2Trash==1.8.0
sentry-sdk==1.16.0
setproctitle==1.3.2
setuptools-git==1.2
shortuuid==1.0.11
six==1.16.0
sklearn==0.0
smmap==5.0.0
soupsieve==2.3.2.post1
SQLAlchemy==2.0.6
stack-data==0.4.0
statsmodels==0.13.5
stevedore==5.0.0
tbats==1.1.2
tenacity==8.2.2
tensorboard==2.11.2
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.11.0
tensorflow-estimator==2.11.0
tensorflow-intel==2.11.0
tensorflow-io-gcs-filesystem==0.31.0
tensornetwork==0.4.6
termcolor==2.2.0
terminado==0.15.0
threadpoolctl==3.1.0
tifffile==2022.10.10
tinycss2==1.1.1
toml==0.10.2
tomli==2.0.1
tomlkit==0.11.4
torch==1.11.0
torchaudio==0.13.1+cu117
torchmetrics==0.11.4
torchvision==0.14.1+cu117
tornado==6.2
tqdm==4.65.0
traitlets==5.3.0
typer==0.4.2
types-Deprecated==1.2.9
typing_extensions==4.4.0
urllib3==1.26.12
wandb==0.12.21
wcwidth==0.2.5
webencodings==0.5.1
Werkzeug==2.2.3
widgetsnbextension==3.6.2
win32-setctime==1.1.0
wrapt==1.14.1
yarl==1.8.2
zipp==3.15.0

**Comment by LubinNK** [comment]
I have this problem too.
